resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -6,10 +6,11 @@
     executor:
       docker:
         image: "resolwebio/archive-samples"
-  version: 0.0.2
+  version: 0.0.3
   type: "data:archive:samples"
   category: other
   persistence: TEMP
+  scheduling_class: interactive
```

Comment:
Let's make this arciver interactive now for the sake of user experience. Let's not make the resolwe archiver interactive. We can implement a better solution later. Wenshould text how long does it take on average to make the archieve in our user workflow. If it's well below 30s, this solutuion may stick for a while.